Packs/Workday/Integrations/WorkdayIAMEventsGenerator/README.md

```diff
@@ -0,0 +1,77 @@
+IAM mock Workday full reports and events.
+
+This integration is for testing propose only.
+To use this integration for testing, take the server with the listening port that has been configured in the integration settings,
+and configure the **Workday_IAM** instance with this url and invoke the *fetch_incidents*.
```

Comment:
looks like you covered almost everything, please just add a note to mark the `longRunning` checkbox as well